DirectX11/HackerContext.cpp

```diff
@@ -1383,12 +1351,7 @@ STDMETHODIMP_(void) HackerContext::GSSetShader(THIS_
 	__in_ecount_opt(NumClassInstances) ID3D11ClassInstance *const *ppClassInstances,
 	UINT NumClassInstances)
 {
-	SetShader<ID3D11GeometryShader, &ID3D11DeviceContext::GSSetShader>
-		(pShader, ppClassInstances, NumClassInstances,
-		 &G->mVisitedGeometryShaders,
-		 G->mSelectedGeometryShader,
-		 &mCurrentGeometryShader,
-		 &mCurrentGeometryShaderHandle);
+	SetShader<ID3D11GeometryShader, &ID3D11DeviceContext::GSSetShader>(pShader, ppClassInstances, NumClassInstances, G->mGeometryShaders);
```

Comment:
This was broken up on separate lines to avoid the need to scroll horizontally.
See this:
![image](https://user-images.githubusercontent.com/379749/78633333-aedfca80-78e4-11ea-80f4-961868fcedaf.png)
I can read it.

See this:
![image](https://user-images.githubusercontent.com/379749/78633349-b8693280-78e4-11ea-8cda-b1b323262b46.png)

I can't read that.